csharp/ql/test/library-tests/controlflow/graph/NodeGraph.expected

```diff
@@ -6600,10 +6600,20 @@ Finally.cs:
 
 #  125| ... / ...
 #-----|  -> Double temp = ...
+#-----| exception(DivideByZeroException) -> catch {...}
```

Comment:
Let's change `System.Math.E` on line 125 in `Finally.cs` to `2.7182818284590451`. Or should we instead include constant values when generating stubs?